web/src/model/scope.ts

```diff
@@ -0,0 +1,77 @@
+import { ContextSingleton } from '../utils/context';
+import { Feature } from './config';
+import { FlowScope } from './flow-query';
+import { TopologyGroupTypes } from './topology';
+
+export type ScopeConfigDef = {
+  id: FlowScope;
+  name?: string;
+  shortName?: string;
+  description?: string;
```

Comment:
are there scope objects that wouldn't have these 3 fields? I think we could make them not optional?
That would allow to get rid of the functions `getScopeName` and `getScopeShortName`

(And I see in `scope-drowndown.tsx` that we call `getScopeName` with a potentially undefined scope but that could be replaced with `scopes.find(sc => sc.id === selected)?.name || t('n/a')`)

Reply:
yeah I'm fine with your suggestion :+1:

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/634/commits/4077304759b101d94236d71a76445171fac45845